lib/plugins/aws/lib/naming.js

```diff
@@ -264,10 +264,16 @@ module.exports = {
   getMethodLogicalId(resourceId, methodName) {
     return `ApiGatewayMethod${resourceId}${this.normalizeMethodName(methodName)}`;
   },
-  getValidatorLogicalId(resourceId, methodName) {
+  getValidatorLogicalId(modelLogicalId) {
+    return `${modelLogicalId}Validator`;
+  },
+  getEndpointValidatorLogicalId(resourceId, methodName) {
```

Comment:
This function can be removed, as it's used now in places where we have `modelLogicalId`, so we can use  `getValidatorLogicalId(modelLogicalId)` instead

Reply:
I added the additional method because there is only 1 validator allowed per API Gateway Method. This was causing issues when using the contentType in the ModelLogicalID. This only becomes an issue when multiple content types are defined in the resource. I have updated to utilize `getModelLogicalId` within this method now to prevent that.